resolwe_bio/processes/import_data/sra_file.py

```diff
@@ -194,6 +204,15 @@ def run(self, inputs, outputs):
         """Run the analysis."""
         fastq_gz = []
         for srr in inputs.sra_accession:
+            if inputs.advanced.prefetch:
+                # prefetch the SRR bundle
+                return_code, _, _ = Cmd["prefetch"][
+                    "-p", inputs.advanced.max_size_prefetch, srr
```

Comment:
Should this option be ` -X` / `--max-size` instead of `-p`?

Reply:
Fixed, `--max-size` parameter options was missing. `-p` prints out the download progress.